chainerx_cc/chainerx/array_repr.cc

```diff
@@ -257,6 +233,7 @@ std::ostream& operator<<(std::ostream& os, const Array& array) {
     // TODO(hvy): We need to determine the output specification of this function, whether or not to align with Python repr specification,
     // and also whether this functionality should be defined in C++ layer or Python layer.
     // TODO(hvy): Consider using a static dimensionality.
+    NoBackpropModeScope scope{};
```

Comment:
I'd like to just make sure that it is not related to the scope of this PR or not?

Reply:
This fix is related to this PR.
https://github.com/chainer/chainer/blob/926e3f9fc058831191eee009d9ffe2d324371b82/chainerx_cc/chainerx/array_repr.cc#L224
Current implementation calls `internal::At` which is one of the routines, but we don't want create backward graph.

Reply:
Makes sense. Thanks for the clarification.